server/server.js

```diff
@@ -4,13 +4,14 @@ const bodyParser = require('body-parser');
 const session = require('express-session');
 const bcrypt = require('bcryptjs');
 const redis = require('connect-redis')(session);
+const kanban = require('./routes/kanban')
 
 
 const PORT = process.env.PORT || 8080;
 const ENV = process.env.NODE_ENV || 'development';
-// const SESSION_SECRET = process.env.SESSION_SECRET
+const SESSION_SECRET = process.env.SESSION_SECRET
 // app.use(session({
-//   store: new redis({ url: 'redis://redis-server:6379', logErrors: true }),
+//   store: new redis({ url: 'redis://redis-server:REDIS_HOST_PORT', logErrors: true }),
```

Comment:
This is good but we want to configure both the Redis URL _and_ port so something like this:

```
store: new redis({ url: `${REDIS_URL}:${REDIS_PORT}`, logErrors: true })
```